modules/apps/headless/headless-discovery/headless-discovery-web/package.json

```diff
@@ -2,7 +2,7 @@
 	"dependencies": {
 		"@clayui/alert": "3.111.0",
 		"@clayui/button": "3.115.0",
-		"@clayui/css": "3.115.2",
+		"@clayui/css": "3.116.0",
```

Comment:
@antonio-ortega warn me about this type of version updates. It seems to be Clay team responsability, so maybe this PR is useless.

Reply:
We constantly update all components on the osb-faro-web side.

Reply:
@interaminense I'd say so however, those are remote apps or samples that may have their own lifecycle, so that decision should probably be made by the corresponding owners. In the case of DXP apps I'd definitely keep things up-to-date, in sync.

Reply:
Yeah, sounds good to me!

Reply:
@dsanz @interaminense Clay team has updated all dependencies, so we don't need to worry about this PR. This is how this `headless-discovery` [package.json](https://github.com/brianchandotcom/liferay-portal/pull/151434/files#diff-a6c75a2b8106fd612f147861da7cda976cf39ac75602280120039aa8d094b1c6) looks now